packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -62,6 +65,64 @@ class TransloaditAssembly extends Emitter {
     this.close()
   }
 
+  #connectServerSentEvents () {
+    this.#sse = new EventSource(`${this.status.websocket_url}?assembly=${this.status.assembly_id}`)
+
+    this.#sse.addEventListener('open', () => {
+      // if server side events works, we don't need websockets anymore (it's just a fallback)
+      if (this.socket) {
+        this.socket.disconnect()
+        this.socket = null
+      }
+      clearInterval(this.pollInterval)
+      this.pollInterval = null
+    })
+
+    /*
+     * The event "message" is a special case, as it
+     * will capture events without an event field
+     * as well as events that have the specific type
+     * other event type.
+     */
+    this.#sse.addEventListener('message', (e) => {
+      if (e.data === 'assembly_finished') {
+        this.#onFinished()
+      }
+
+      if (e.data === 'assembly_uploading_finished') {
+        this.emit('executing')
+      }
+
+      if (e.data === 'assembly_upload_meta_data_extracted') {
+        this.emit('metadata')
+        this.#fetchStatus({ diff: false })
+      }
+    })
+
+    this.#sse.addEventListener('assembly_upload_finished', (e) => {
+      const file = e.data
+      this.emit('upload', file)
+      this.status.uploads.push(file)
+    })
+
+    this.#sse.addEventListener('assembly_result_finished', (e) => {
+      const [stepName, result] = JSON.parse(e.data)
+      this.emit('result', stepName, result)
```

Comment:
Shouldn't it be this?
```suggestion
      this.emit('transloadit:result', stepName, result)
```

Reply:
https://github.com/transloadit/uppy/blob/5ec7187825c6af473a178a37196a5636c74072c5/packages/@uppy/transloadit/src/Assembly.js#L103-L109